README.md

```diff
@@ -12,20 +12,45 @@ Offers alternative types of file parameter that are compatible with Pipeline and
 
 See [JENKINS-27413](https://issues.jenkins-ci.org/browse/JENKINS-27413) and [JENKINS-29289](https://issues.jenkins-ci.org/browse/JENKINS-29289) for background.
 
+## Usage in declarative pipeline
+
+You can now declare file parameters as is in declarative pipeline:
+
+```
+pipeline {
+  agent any
+  parameters {
+    base64File(name: 'FILE')
+    stashed64File(name: 'FILE-STASH')
+  }
+  stages {
+    stage('Example') {
+      steps {
+        withFileParameter('FILE') {
```

Comment:
IIRC Declarative supports using block-scoped steps in an `option` block. Does it work here?

Reply:
yep it works

Reply:
But you prefer to demonstrate using a block-scoped step inside `steps`?

Reply:
uhm I missed that one but I'm not sure what sort of syntax you mean?

Reply:
Bug in Declarative it seems: https://issues.jenkins.io/browse/JENKINS-64761